### DISCUSSION TITLE
OpenShift web console developer "+Add" creates Route instead of Ingress

### DISCUSSION THREAD

**Discussion by arska** [appuio/appuio-cloud-community#32] [Help Needed]
As a developer using the OpenShift web console developer mode
When using the "+Add" function (https://console.cloudscale-lpg-2.appuio.cloud/add/ns/aarno-playground), choosing "deploy image" with my image "ghcr.io/arska/requestdump:master", choosing "show advanced route options", custom hostname "requestdump.vshn.ch", enabling "secure route", TLS termination edge, Insecure traffic redirect, then clicking "create"
Then a Route object is created, leading to the letsencrypt.net integration not working (https://docs.appuio.cloud/user/how-to/getting-a-certificate.html)

Since the OCP web console creates Route objects by default this feels like a usability bug

**Comment by tobru** [comment]
Unfortunately we have no control over this feature, that's part of the OpenShift Web Console. And cert-manager has no support for the `Route` object and doesn't plan to add it (https://github.com/jetstack/cert-manager/issues/1064).
